algosdk/dryrun_results.py

```diff
@@ -63,13 +63,13 @@ def attrname(field):
     def app_call_rejected(self) -> bool:
         return (
             False
-            if self.app_call_messages is None
-            else "REJECT" in self.app_call_messages
+            if self.app_call_messages is None  # type: ignore  # dynamic attribute
+            else "REJECT" in self.app_call_messages  # type: ignore  # dynamic attribute
```

Comment:
@jdtzmn Leaving a request that applies to multiple places in the PR.  

Rather than ignore all errors (`# type: ignore`), can you change to ignore _only_ the alerting error (suggestion shows example)?

As discussed in https://mypy.readthedocs.io/en/stable/error_codes.html#error-codes, ignoring specific errors minimizes the chance that we'll unintentionally ignore future errors on the same line.

```suggestion
            if self.app_call_messages is None  # type: ignore[attr-defined]
            else "REJECT" in self.app_call_messages  # type: ignore[attr-defined]
```